modules/database/src/admin/index.ts

```diff
@@ -305,6 +316,7 @@ export class AdminHandlers {
         {
           path: '/schemas/:schemaId/extensions',
           action: ConduitRouteActions.POST,
+          description: `Sets extension fields to a schema.`,
```

Comment:
This one actually sets, updates and deletes the `Database` module's own extension** for that schema.
You could phrase this like so:
`Sets a Database-owned extension for target schema, expanding it with additional fields. Passing an empty fields object removes the extension.`